src/res/osscameroon_issues.json

```diff
@@ -595,6 +595,12 @@
     "author": "elhmn",
     "labels": []
   },
+  {
+    "issue": "https://github.com/osscameroon/miniyotas/issues/26",
+    "title": "Dockerise miniyotas",
+    "author": "elhmn",
+    "labels": []
+  },
```

Comment:
This file is automatically generated, and should not be modified manually

Reply:
Alright, so what am i expected to do with the file please, should i remove line 598-603

Reply:
Yes those changes should not be part of the PR, so removing those lines would be great

Reply:
alright